app/assets/javascripts/class_edit.js

```diff
@@ -19,12 +19,12 @@ function add_puppet_class(item){
   var content = $(item).closest('li').clone();
   content.attr('id', 'selected_puppetclass_'+ id);
   content.append("<input id='" + type +"_puppetclass_ids_' name='" + type +"[puppetclass_ids][]' type='hidden' value=" +id+ ">");
-  content.children('span').tooltip();
 
   var link = content.children('a');
   var links = content.find('a');
+  var fullname = _.last(link.data('original-title').split(' '))
```

Comment:
"Click to add %s", which could be localized so replacing add with remove won't work.